packages/calcite-components/src/components/sheet/resources.ts

```diff
@@ -11,5 +11,5 @@ export const CSS = {
   height: "height",
 };
 
-export const sheetResizeStep = 10;
-export const sheetResizeShiftStep = 25;
+export const resizeStep = 10;
```

Comment:
Nitpick: Could these be combined with the Shell Panel definitions in a shared resource file? Do we want to have these be the same for Dialog?

Reply:
I think we should open a new issue/discussion for these. I just picked these values a while back. We could move them to a common resources file though. It would also apply to the dialog component which have different values currently.

Reply:
updated!